### PR TITLE
Feature/6218 skipped integration tests 2

### PR DESCRIPTION
**Pull request by clarissacastella**

### CODE REVIEW

File: app/controllers/test_controller.rb [hunk 1]
```diff
@@ -74,6 +93,28 @@ def new_source
     render_success 'project_source', ps
   end
 
+  def media_status
+    require 'sidekiq/testing'
+    Sidekiq::Testing.inline!
```
Comment (caiosba) [inline app/controllers/test_controller.rb]: If you really need Sidekiq here to run inline, please use it as a block, so it doesn't after other parts of the code. Like this:
```
Sidekiq::Testing.inline! do
   pm = ProjectMedia.find(params[:pm_id])
   s = pm.annotations.where(annotation_type: 'status').last.load
   s.status = params[:status]
   s.save!
   render_success 'project_media', pm
end
```
More at https://github.com/mperham/sidekiq/wiki/Testing (look for "Each of the above methods also accepts a block").

File: app/controllers/test_controller.rb [hunk 2]
```diff
@@ -74,6 +93,28 @@ def new_source
...
+    pm = ProjectMedia.find(params[:pm_id])
+    s = pm.annotations.where(annotation_type: 'status').last.load
+    s.status = params[:status]
+    s.save!
+    render_success 'project_media', pm
+  end
+
+  def new_media_tag
+    require 'sidekiq/testing'
+    Sidekiq::Testing.inline!
```
Comment (caiosba) [inline app/controllers/test_controller.rb]: Same as comment above.

**Comment by caiosba** [comment]
We need tests for the `TestController` methods two :)

They should be here: https://github.com/meedan/check-api/blob/develop/test/controllers/test_controller_test.rb

Check the examples there for reference.

**Comment by clarissacastella** [comment]
Caio's comments addressed.
Please review again.

**Comment by caiosba** [comment]
Looks good! If all tests pass and the coverage is 100%, feel free to merge.

**Pull request merged**